cidc_schemas/schemas/templates/metadata/ihc_template.json

```
@@ -67,21 +67,26 @@
                 },
                 "data_columns": {
                     "Sample": {
-                        "cimac ID": {
+                        "cimac id": {
                             "merge_pointer": "0/cimac_id",
                             "type_ref": "sample.json#properties/cimac_id"
                         },
                         "IHC image": {
                             "merge_pointer": "0/files/ihc_image",
                             "is_artifact": 1,
-                            "gcs_uri_format": "{protocol identifier}/ihc/{cimac id}/ihc.tiff",
+                            "gcs_uri_format": {
+                                        "format": "lambda val, ctx: ctx['protocol identifier'] + '/ihc/' + ctx['cimac id'] + '/ihc_image.'.format_map(ctx) + val.rsplit('.', 1)[-1]",
```

no need for `.format_map` if you're already manually getting vars from `ctx` (`ctx['protocol identifier']`) and concatenating them.
```suggestion
                                        "format": "lambda val, ctx: ctx['protocol identifier'] + '/ihc/' + ctx['cimac id'] + '/ihc_image.' + val.rsplit('.', 1)[-1]",
```